src/main/java/edu/illinois/cs/dt/tools/minimizer/MinimizerPlugin.java

```diff
@@ -189,7 +189,9 @@ public MinimizerPlugin(final InstrumentingSmartRunner runner) {
         return Stream.of(tm);
     }
 
-    private List<String> reorderOriginalOrder(List<String> intended, List<String> originalOrder) {
+    private List<String> reorderOriginalOrder(List<String> failingOrder, List<String> originalOrder, String dependentTest) {
+        List<String> intended = new ArrayList<>(failingOrder);
```

Comment:
I assume the issue is that the dependentTest is not always found in the passed in order, so it should be added in just in case? If it is already there, should there be a check beforehand? Also, why create a new ArrayList around the passed in failingOrder and then have a new ArrayList retList wrap around that one after adding in the dependentTest? Why not add it in after "List<String> retList = new ArrayList<>(intended);" (and then you can keep the name intended as the parameter.

Reply:
> I assume the issue is that the dependentTest is not always found in the passed in order, so it should be added in just in case?
Yes, it is. I've changed the code as you suggested.